### PR TITLE
Adds fallback address for debian releases 

### PR DESCRIPTION
**Pull request by gurkanindibay**
Recently, we have problems to download debian/bookworm 
packages from packagecloud, since packagecloud repository 
config file does not work with release number but works 
with release code name. 
The one that is not working
https://packagecloud.io/install/repositories/citusdata/community/config_file.list?os=debian&dist=12&source=script
The one that is working
https://packagecloud.io/install/repositories/citusdata/community/config_file.list?os=debian&dist=bookworm&source=script

There are two fixes in this PR to handle this problem
1. Adds new debian releases in the codename matrix for debian
2. Adds fallback address for debian releases which 
includes code names instead of release numbers.

### CODE REVIEW

File: community/deb.sh [hunk 1]
```diff
@@ -249,9 +258,15 @@ main ()
 
   echo -n "Installing $apt_source_path... "
 
-  # create an apt config file for this repository
```
Comment (hanefi) [inline community/deb.sh]: Why did we remove this comment?

File: community/deb.sh [hunk 2]
```diff
@@ -249,9 +258,15 @@ main ()
...
   curl -sSf "${apt_config_url}" > $apt_source_path
   curl_exit_code=$?
+  if [ "${os}" = "debian" ] && [ "$curl_exit_code" -ne "0" ]; then
```
Comment (hanefi) [inline community/deb.sh:277]: Can we add a comment before this conditional to explain why we need such a change?

**Review by hanefi — CHANGES_REQUESTED** [review]
After addressing the comments, can you apply the same patch to other 3 `deb.sh` files as well?

**Review by hanefi — DISMISSED** [review]

File: community/deb.sh (hunk 1 above)
Comment (gurkanindibay) [reply community/deb.sh]: Removed mistakenly. Added again

**Review by hanefi — APPROVED** [review]

**Pull request merged**